src/etc/roles/literal/termRole.js

```diff
@@ -14,7 +14,7 @@ const termRole: ARIARoleDefinition = {
   relatedConcepts: [
     {
       concept: {
-        name: 'dfn',
+        name: 'dt',
```

Comment:
`dfn` **and** `dt` map to `term`: 
- [`dfn` mapping](https://www.w3.org/TR/html-aam-1.0/#details-id-29)
- [`dt` mapping](https://www.w3.org/TR/html-aam-1.0/#details-id-33)

Reply:
I didn't look for `dfn` on specification but reading the `README.md` I thought was correct the `definition` role

`'{"name": "dfn"}' => Set { 'definition' },`

Reply:
> but reading the README.md

Right, we should update that one as well